tests/cases/fourslash/documentHighlightInExport1.ts

```diff
@@ -0,0 +1,11 @@
+﻿/// <reference path='fourslash.ts'/>
+
+// @Filename: file1.ts

```

Comment:
File name isn't needed unless you're doing a test with multiple files, or an unusual file extension.

Reply:
Neat.